### PR TITLE
Remove `ON CONFLICT DO NOTHING` clause in e2e custom sqls

### PR DESCRIPTION
**Pull request by pkong-ds**
I found the issue was not

> error not thrown

Instead, it is

> database did not throw error when inserting duplicate primary key records

Which is caused by `ON CONFLICT DO NOTHING` clause in custom sql.

I will remove all `ON CONFLICT DO NOTHING` clauses as a fix 🙏 Since UUID() will be introduced in [DEV-1765](https://linear.app/authgear/issue/DEV-1765/add-generateuuid-to-e2e-template)

### CODE REVIEW

File: e2e/pkg/testrunner/testcase.go [hunk 1]
```diff
@@ -95,28 +98,29 @@ func (tc *TestCase) Run(t *testing.T) {
...
 				t.Errorf("failed to execute custom SQL: %v", err)
-				return false
+				return
 			}
 		default:
-			t.Errorf("unknown before hook type: %s", beforeHook.Type)
-			return false
+			errStr := fmt.Sprintf("unknown before hook type: %s", beforeHook.Type)
+			t.Errorf(errStr)
+			return errors.New(errStr)
```
Comment (louischan-oursky) [inline e2e/pkg/testrunner/testcase.go:119]: err := fmt.Errorf("unknown before hook type: %v", beforeHook.Type)
t.Errorf("%v", err)
return err

File: e2e/pkg/testrunner/testcase.go [hunk 2]
```diff
@@ -65,8 +66,9 @@ func (tc *TestCase) Run(t *testing.T) {
 		return
 	}
 
-	ok := tc.executeBeforeAll(t, cmd)
-	if !ok {
+	err = tc.executeBeforeAll(t, cmd)
+	if err != nil {
+		t.Errorf("failed to execute before hooks: %v", err)
```
Comment (louischan-oursky) [inline e2e/pkg/testrunner/testcase.go:71]: Remove this line.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

File: e2e/pkg/testrunner/testcase.go [hunk 3]
```diff
@@ -95,28 +98,29 @@ func (tc *TestCase) Run(t *testing.T) {
 }
 
 // Execute before hooks to prepare fixtures
-func (tc *TestCase) executeBeforeAll(t *testing.T, cmd *End2EndCmd) (ok bool) {
+func (tc *TestCase) executeBeforeAll(t *testing.T, cmd *End2EndCmd) (err error) {
```
Comment (louischan-oursky) [inline e2e/pkg/testrunner/testcase.go:101]:
```
func (tc *TestCase) executeBeforeAll(cmd *End2EndCmd) (err error) {
  if err != nil {
    return fmt.Errorf("asdasd: %w", err)
  }
}
```

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

**Comment by pkong-ds** [comment]
@louischan-oursky @IniZio updated, thanks!

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**